v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dbutils/dml/CassandraTypeHandler.java

```diff
@@ -0,0 +1,940 @@
+   * <p>This method checks if the {@code integerValue} is within the valid range for a {@code
+   * smallint} (i.e., between {@link Short#MIN_VALUE} and {@link Short#MAX_VALUE}). If the value is
+   * out of range, it throws an {@link IllegalArgumentException}.
+   *
+   * @param integerValue The integer value to be converted.
+   * @return The converted {@code short} value.
+   * @throws IllegalArgumentException If the {@code integerValue} is out of range for a {@code
+   *     smallint}.
+   */
+  private static short convertToSmallInt(Integer integerValue) {
+    if (integerValue < Short.MIN_VALUE || integerValue > Short.MAX_VALUE) {
+      throw new IllegalArgumentException("Value is out of range for smallint.");
+    }
+    return integerValue.shortValue();
+  }
+
+  /**
+   * Converts an {@link Integer} to a {@code byte} (TinyInt).
+   *
+   * <p>This method checks if the {@code integerValue} is within the valid range for a {@code
+   * tinyint} (i.e., between {@link Byte#MIN_VALUE} and {@link Byte#MAX_VALUE}). If the value is out
+   * of range, it throws an {@link IllegalArgumentException}.
+   *
+   * @param integerValue The integer value to be converted.
+   * @return The converted {@code byte} value.
+   * @throws IllegalArgumentException If the {@code integerValue} is out of range for a {@code
+   *     tinyint}.
+   */
+  private static byte convertToTinyInt(Integer integerValue) {
+    if (integerValue < Byte.MIN_VALUE || integerValue > Byte.MAX_VALUE) {
+      throw new IllegalArgumentException("Value is out of range for tinyint.");
+    }
+    return integerValue.byteValue();
+  }
+
+  /**
+   * Escapes single quotes in a Cassandra string by replacing them with double single quotes.
+   *
+   * <p>This method is commonly used to sanitize strings before inserting them into Cassandra
+   * queries, where single quotes need to be escaped by doubling them (i.e., `'` becomes `''`).
+   *
+   * @param value The string to be escaped.
+   * @return The escaped string where single quotes are replaced with double single quotes.
+   */
+  private static String escapeCassandraString(String value) {
+    return value.replace("'", "''");
+  }
+
+  /**
+   * Converts a string representation of a timestamp to a Cassandra-compatible timestamp.
+   *
+   * <p>The method parses the {@code value} as a {@link ZonedDateTime}, applies the given timezone
+   * offset to adjust the time, and converts the result into a UTC timestamp string that is
+   * compatible with Cassandra.
+   *
+   * @param value The timestamp string in ISO-8601 format (e.g., "2024-12-05T10:15:30+01:00").
+   * @param timezoneOffset The timezone offset (e.g., "+02:00") to apply to the timestamp.
+   * @return A string representation of the timestamp in UTC that is compatible with Cassandra.
+   * @throws RuntimeException If the timestamp string is invalid or the conversion fails.
+   */
+  private static String convertToCassandraTimestamp(String value, String timezoneOffset) {
```

Comment:
This method is set to be private and I see no usage for it in the class, same for other method with the same name

Reply:
Same comment for multiple other private methods

Reply:
I think you need to revisit the access modifiers, method like `handleCassandraGenericDateType` should be `public` and utility methods like `convertToCassandraTimestamp` should be `private`

Reply:
Yes @shreyakhajanchi as I mentioned already we are revisiting the access modifiers in DML PR as we have one method which is going to handle all case so rest all will be private except one mapping method